src/series.jl

```diff
@@ -21,32 +21,44 @@ handlemissings(s::Surface) = Surface(handlemissings(s.surf))
 handlemissings(v::Volume) = Volume(handlemissings(v.v), v.x_extents, v.y_extents, v.z_extents)
 
 # default: assume x represents a single series
-convertToAnyVector(x) = Any[prepareSeriesData(x)]
+convertToAnyVector(x, plotattributes) = Any[prepareSeriesData(x)]
 
 # fixed number of blank series
-convertToAnyVector(n::Integer) = Any[zeros(0) for i in 1:n]
+convertToAnyVector(n::Integer, plotattributes) = Any[zeros(0) for i in 1:n]
 
 # vector of data points is a single series
-convertToAnyVector(v::AVec{<:DataPoint}) = Any[prepareSeriesData(v)]
+convertToAnyVector(v::AVec{<:DataPoint}, plotattributes) = Any[prepareSeriesData(v)]
 
 # list of things (maybe other vectors, functions, or something else)
-convertToAnyVector(v::AVec) = vcat((convertToAnyVector(vi) for vi in v)...)
+function convertToAnyVector(v::AVec, plotattributes)
+    if all(x -> isa(x, Number) || ismissing(x), v) || all(x -> isa(x, AbstractString) || ismissing(x), v)
+        convertToAnyVector(convert.(DataPoint, v), plotattributes)
```

Comment:
`convert.(DataPoint, v)` is not always strongly typed: e.g. `convert.(Plots.DataPoint, Any[1,2.3,missing])` gives a vector of `Any`, so it creates an infinite recursion here.

Reply:
Oh, was looking at an outdated version, and didn't notice this was already merged.